extensions/spotify/init.lua

```diff
@@ -125,4 +153,54 @@ function spotify.getCurrentTrack()
     return tell('name of the current track')
 end
 
+--- hs.spotify.getPlayerState()
+--- Function
+--- Gets the current state of Spotify
+---
+--- Parameters:
+---  * None
+---
+--- Returns:
+---  * A string containing one of hs.spotify.state_stopped, hs.spotify.state_paused, hs.spotify.state_playing
+function spotify.getPlayerState()
+   return tell('get player state')
+end
+
+--- hs.spotify.isRunning()
+--- Function
+--- Returns whether Spotify is currently open. Most other functions in hs.spotify will automatically start the application, so this function can be used to guard against that.
+---
+--- Parameters:
+---  * None
+---
+--- Returns:
+---  * A boolean value indicating whether the Spotify application is running.
+function spotify.isRunning()
+   return hs.application.get("Spotify") ~= nil
```

Comment:
By convention (and not properly documented anywhere, so this one is absolutely my fault) we don't call other modules with absolute names - it relies on the automatic module loading, which users may have disabled. Instead, add a `local application = require "hs.application"` at the top of the file, and use `application.get(blah)`